sys-utils/eject.c

```diff
@@ -807,7 +807,7 @@ static int umount_partitions(struct eject_control *ctl)
 	return count;
 }
 
-static int is_hotpluggable(const struct eject_control *ctl)
+static int is_removable(const struct eject_control *ctl)
 {
```

Comment:
I'd suggest is_ejectable() and test for hotplug and removable.

Reply:
Done